### PR TITLE
New ringtones

### PR DESCRIPTION
**Pull request by BonniePandora**
Adds in a few new tones for PDA ringers, mostly engineering/technical based.

There's now "dust", "rad", "servo" and "spark" alongside all the existing ones, figured a little bit of variety wouldn't do any harm.

### CODE REVIEW

File: code/modules/pda/pda.dm [hunk 1]
```diff
@@ -34,7 +34,11 @@ var/global/list/obj/item/device/pda/PDAs = list()
 								"honk" = 'sound/items/bikehorn.ogg',
 								"SKREE" = 'sound/voice/shriek1.ogg',
 								// "holy" = 'sound/items/PDA/ambicha4-short.ogg',
-								"xeno" = 'sound/voice/hiss1.ogg')
+								"xeno" = 'sound/voice/hiss1.ogg',
+								"dust" = 'sound/effects/supermatter.ogg',
+								"spark" = 'sound/effects/spark4.ogg',
```
Comment (Very-Soft) [inline code/modules/pda/pda.dm:39]: should be sparks

File: code/modules/pda/pda.dm [hunk 2]
```diff
@@ -34,7 +34,11 @@ var/global/list/obj/item/device/pda/PDAs = list()
 								"honk" = 'sound/items/bikehorn.ogg',
 								"SKREE" = 'sound/voice/shriek1.ogg',
 								// "holy" = 'sound/items/PDA/ambicha4-short.ogg',
-								"xeno" = 'sound/voice/hiss1.ogg')
+								"xeno" = 'sound/voice/hiss1.ogg',
+								"dust" = 'sound/effects/supermatter.ogg',
+								"spark" = 'sound/effects/spark4.ogg',
+								"rad" = 'sounds/items/geiger/high1.ogg',
```
Comment (Very-Soft) [inline code/modules/pda/pda.dm:40]: should be sound

Comment (BonniePandora) [reply code/modules/pda/pda.dm]: Fixed in the new commit

File: code/modules/pda/pda.dm (hunk 1 above)
Comment (BonniePandora) [reply code/modules/pda/pda.dm]: This is fixed now too

**Pull request merged**